flatdata-cpp/include/flatdata/MultiVector.h

```diff
@@ -20,10 +20,15 @@ namespace flatdata
  *    MultiVector< 32, A, C, B > vector = ...;
  *    for ( uint64_t index = 0; index < num_items; index++ )
  *    {
- *        vector.add_to_current_item< A >( A( ... ) );
- *        vector.add_to_current_item< A >( A( ... ) );
- *        vector.add_to_current_item< B >( B( ... ) );
- *        vector.add_to_current_item< C >( C( ... ) );
+ *        auto list = vector.grow( );
+ *        auto a = list.add< A >( );
+ *        fill_data_a( a );
+ *        auto another_a = list.add< A >( );
+ *        fill_data_a( another_a );
+ *        auto c = list.add< B >( );
+ *        fill_data_b( b );
+ *        auto d = list.add< C >( );
+ *        fill_data_c( c );
  *        vector.next_item( );
```

Comment:
I think this should be `grow`. But as far as I understand, this is not needed at all anymore, correct?